### ISSUE TITLE
FORM Cobyla without failure - no warning or error message

### ISSUE THREAD

**Issue by regislebrun** [openturns/openturns#1253]
I made a little test :
  *  g = r-s with 2 uniform distributions without any common point r U(2,3) and s U(0,1)
  *  I use the Cobyla algorithm 

I hoped that OpenTURNS would insult me to perform that kind of analysis, but no... the calculus ends without any message and gives me a result.

It could be useful to add another criteria to identify the good convergence of Cobyla ?

Linked to [http://trac.openturns.org/ticket/391](http://trac.openturns.org/ticket/391)

**Comment by mbaudin47** [comment]
This should be solved after #2472.

**Comment by adutfoy** [comment]
Now, the library explains that the result is not reliable.
```
from openturns import *

inputDist = JointDistribution([Uniform(2,3), Uniform(0,1)])
fct = SymbolicFunction(['r', 's'], ['r-s'])
output = CompositeRandomVector(fct, RandomVector(inputDist))

event = ThresholdEvent(output, Less(), 0.0)
solver = Cobyla()
algo = FORM(solver, event, inputDist.getMean())
algo.run()
result = algo.getResult()

```

We get: 
RuntimeError: Exception : Obtained design point is not on the limit state: its image by the limit state function is 1, which is incompatible with the threshold: 0 considering the limit state tolerance of the optimization algorithm: 1e-05
